ntoskrnl/ke/i386/mproc.c

```diff
@@ -48,7 +49,7 @@ KeStartAllProcessors(VOID)
         DPCStack = NULL;
 
         // Allocate structures for a new CPU.
-        APInfo = ExAllocatePoolZero(NonPagedPool, sizeof(APINFO), '  eK');
+        APInfo = ExAllocatePoolZero(NonPagedPool, sizeof(*APInfo), TAG_KERNEL);
```

Comment:
Why are you doing sizeof like this? does it make a difference?

Reply:
in the hypothetical case that the variable would ever be changed in type, Serges might intend that it auto-adopts to that, I guess.